Src/Plugins/Security/SEC101_021.DropboxAppCredentialsValidator.cs

```diff
@@ -48,16 +47,14 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                                                                 IDictionary<string, string> options,
                                                                 ref ResultLevelKind resultLevelKind)
         {
-            const string uri = "https://content.dropboxapi.com/2/files/get_thumbnail_v2";
-
             string id = fingerprint.Id;
             string secret = fingerprint.Secret;
             string credentials = Convert.ToBase64String(Encoding.ASCII.GetBytes(string.Format("{0}:{1}", id, secret)));
             HttpClient httpClient = CreateOrRetrieveCachedHttpClient();
 
             try
             {
-                using var request = new HttpRequestMessage(HttpMethod.Post, uri);
+                using var request = new HttpRequestMessage(HttpMethod.Post, Uri);
                 request.Headers.Authorization = new AuthenticationHeaderValue("Basic", credentials);
                 request.Headers.Add("Dropbox-API-Arg", @"{""resource"": {"".tag"": ""path"",""path"": ""/a.docx""},""format"": ""jpeg"",""size"": ""w64h64"",""mode"": ""strict""}");
 
```

Comment:
you could encapsulate this in a method and re-use everything in the test just like the other validators.